use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,227 @@
+# Init process for Fluent Bit on ECS, multi-config support
+
+
+
+### What is init process and multi-config support?
+
+Multi-config support means when you use a Fluent Bit image with `init` tag on ECS, you can use some config files conveniently to configure Fluent Bit. These config files can be your own files that you upload to S3, or you can use the [built-in config files](https://github.com/aws/aws-for-fluent-bit/tree/mainline/ecs) we provide directly. Init process achieves this feature. 
+
+
+
+### Why we create init process?
+
+Currently, if you want to set up additional configs for Fluent Bit running on ECS, you need to [build a custom image](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file). And you cannot set up a [PARSER] section for Fluent Bit because using a parser in Fluent Bit requires you to change the Fluent Bit command or specify the path of the parser file under the [SERVICE] section.
+
+Init process is designed to solve above mentioned problems and improve user experience of using Fluent Bit on ECS. You only need to set the `ARN` or `Path` of the config file in the ECS Task Definition, without telling us if it is a parser file. Init process will process each config file, use [@INCLUDE](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/configuration-file#config_include_file-1) keyword to add them to the main config file, and change the Fluent Bit command if the parser is used.
```

Comment:
" without telling us if it is a parser file" => "without telling us if it is a parser file or normal config file".